bigtable/api/BigtableTest/runTests.ps1

```diff
@@ -12,15 +12,17 @@
 # License for the specific language governing permissions and limitations under
 # the License.
 
-Import-Module ..\..\..\BuildTools.psm1 -DisableNameChecking
+# Skipping tests because of https://github.com/GoogleCloudPlatform/dotnet-docs-samples/issues/992
 
-Require-Platform Win*
-Set-TestTimeout 600
+# Import-Module ..\..\..\BuildTools.psm1 -DisableNameChecking
```

Comment:
Is there a way of just returning immediately, so it becomes a two-line change in each direction?